draft-ietf-ppm-dap.md

```diff
@@ -526,6 +528,11 @@ Report share:
   The report share also includes some associated data used for processing the
   report.
 
+Task:
+: A measurement task in which measurements of an understood type will be
```

Comment:
Low confidence nit
```suggestion
: An aggregation task in which measurements of an understood type will be
```

Reply:
We use the phrase "measurement task" in a couple other places in the doc, so there's precedent there, but now that I look at this again, I don't like using "measurement" twice in this sentence.